### PR TITLE
Updated the README.md

### PR DESCRIPTION
**Pull request by Nikhilbabup**
I updated README.md for security-bc based on what I had to do locally to get the integration tests to run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +57,10 @@ npm run start:authentication-svc
 # start the authorization service
 npm run start:authorization-svc 
 ```
+
+To run those services locally, you need to pass 2 env vars like this (executed in modules/authentication-svc):
+
+```bash
+export PRIVATE_CERT_PEM_FILE_PATH=test_keys/private.pem
+export IAM_STORAGE_FILE_PATH=dist/authN_TempStorageFile
```
Comment (koekiebox) [inline README.md:58]: Also include a troubleshoot section for the SSL issue:
image

Details here:
https://github.com/auth0/node-jsonwebtoken/issues/826

Comment (koekiebox) [reply README.md:58]: @Nikhilbabup Are all the integration tests passing?

**Review by koekiebox — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Nikhilbabup) [reply README.md:58]: @koekiebox Yes all the integration tests passing

Comment (Nikhilbabup) [reply README.md:58]: Screenshot from 2022-10-05 16-24-26

Comment (koekiebox) [reply README.md:58]: Perfect.

File: README.md [hunk 2]
```diff
@@ -57,3 +57,18 @@ npm run start:authentication-svc
...
+```
+
+## Troubleshoot 
+
+### Unable to load dlfcn_load
+```bash
+error:25066067:DSO support routines:dlfcn_load:could not load the shared library
+```
```
Comment (koekiebox) [inline README.md:72]: Please include the full URL to the token issue resolution: 
https://github.com/auth0/node-jsonwebtoken/issues/826
`https://github.com/auth0/node-jsonwebtoken/issues/826`

**Review by koekiebox — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (Nikhilbabup) [reply README.md:72]: I included the full URL to the token issue resolution
Screenshot from 2022-10-06 09-57-56

**Review by pedrosousabarreto — APPROVED** [review]
Thanks

**Pull request merged**